### PR TITLE
Inserting analytics to .html files (DO NOT DELETE)

### PR DESCRIPTION
**Pull request by AnmAtAnm**
[Not for commit. Just looking for review.]

Inserting Google Analytics tracking to .html files that are not simple redirects.

This commit branches off the last master release. The branch can be rebased off of
future master push commits as a means to insert the analytics prior to updating
the blockly-demo AppSpot domain.

When doing so, verify there are no new .html files with the following command:

```
$ find . -type f -name "*.html" | xargs grep -L GoogleAnalyticsObject
```

The following are expected results:
```
./appengine/redirect.html
./demos/interpreter/index.html
```

### CODE REVIEW

**Comment by picklesrus** [comment]
Are these code snippets all the same? I spot checked a couple and it seems like they are.  Could we put that snippet in a file and just include it rather than copy/pasting?

**Comment by AnmAtAnm** [comment]
Yes, they are all the same. Analytics stuff like this are usually included inline for performance reasons and ensuring network requests don't delay timing logs, or similar side effects. I guess in our case the impact is minimal, but if I had to inject anything (such as the `<script>` with `src`), it seemed just as easy to inject the whole script (as recommended).

**Comment by NeilFraser** [comment]
If you care, there are faster approaches to load Analytics:
  https://neil.fraser.name/news/2010/04/07/
Although the Analytics API has changed over the last seven years and that code isn't literally correct any more, the concept still applies.

Though I can't imagine why one would want stats on the test and demo pages.  Feels rather intrusive since many copies of these are running on developer's own computers.

**Comment by picklesrus** [comment]
@NeilFraser This won't get submitted.  This is just a template that we'd use to include the js blobs only when pushing the demos to blockly-demos.appspot.com.

The copy/paste seems silly but I suppose it doesn't change much so don't really care all that much.

**Comment by AnmAtAnm** [comment]
Details: https://developers.google.com/analytics/devguides/collection/analyticsjs/

File: tests/mocha/index.html [hunk 1]
```diff
@@ -3,7 +3,14 @@
   <head>
     <meta charset="utf-8">
     <title>Mocha Tests for Blockly</title>
-
+    <script>
```
Comment (AnmAtAnm) [inline tests/mocha/index.html]: Test files like this shouldn't have the analytics.  Please revert this commit.

Comment (AnmAtAnm) [reply tests/mocha/index.html]: Also, update the docs to include this file in the excluded list.

**Pull request closed without merging**